collect-tips.rst

```diff
@@ -0,0 +1,7 @@
+**************************
+Best Practices and Tips
+**************************
+
+- Specifying the GPS requirements can help users decide the accuracy of location they send and hence help save battery. For more information on enabling GPS services and its relation with battery life visit `this site <https://support.google.com/nexus/answer/3467281?hl=en>`_.
```

Comment:
This conflates GPS (a location technology) with location. I think this could be a lot more precise. I'd start with "You can configure your device's location mode based on accuracy, speed, and battery use. For more information on location mode and it's relationship to battery life, visit..."

Reply:
We should also note that for most deployments, you'll want to use "High accuracy" true GPS.